README.md

```diff
@@ -0,0 +1,130 @@
+## 📚 Book Store API
+
+<p style="text-align: center;">
+  <img src="src/img/title.png" alt="Book Store API" style="width: 512px;"/>
+</p>
+
+Welcome to the Book Store API! This project serves as the backbone for an online bookstore, offering essential functionalities for searching, browsing, and purchasing books. Inspired by the need for a robust and efficient solution, the Book Store API aims to tackle the challenges faced by online book retailers, providing a seamless experience for both users and administrators.
+
+## 🧱 Structure
+- [👉 Technologies Used](#-technologies-and-tools-used)
+- [👉 Features](#-features-and-functionality)
+- [👉 Installation](#-installation)
+- [👉 Postman Collection](#-postman-collection)
+- [👉 Video-overview](#-video-overview)
+
+## 🛠️ Technologies and Tools Used
+
+- **Spring Boot**: For building the application framework.
+- **Spring Security**: To handle authentication and authorization.
+- **Spring Data JPA**: For database interactions using JPA.
+- **Spring Web**: For building web-based applications with Spring MVC.
+- **Swagger**: For API documentation and testing.
+- **Hibernate**: As the ORM tool.
+- **MySQL**: For the relational database.
+- **Liquibase**: For database schema versioning and management.
+- **Docker**: For containerizing the application.
+- **JUnit**: For unit testing.
+- **Mockito**: For mocking in tests.
+- **Maven**: For project management and dependency management.
+
+## 🚀 Features and Functionality
+
+### Authentication
+
+- **Register user**: `POST /api/auth/register`
+- **Login user**: `POST /api/auth/login`
```

Comment:
this section is quite important we can expand it to explain things more about how your auth is working in the app, leave a request/response examples, how to use the token the we receive and leave credentials to simple and admin users, so that the visitor of your project could use them for testing purposes